specs/subresourceintegrity/spec.markdown

```diff
@@ -243,7 +243,7 @@ request's [integrity metadata][], and MAY support additional hash functions.
 #### Agility
 
 Multiple sets of [integrity metadata][] may be associated with a single
-resource in order to provide agility in the face of future discoveries.
+resource in order to provide agility in the face of future cryptographic developments.
```

Comment:
I am not sure about this change; breaking a hash function is not quite a cryptographic development as much as a discovery. but don't really care.